viewing_party/main.py

```diff
@@ -0,0 +1,135 @@
+# wave 1
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title == None or genre == None or rating == None:
+        new_movie = None
+    else:
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched":[
+            movie
+        ]
+        }
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist":[
+            movie
+        ]
+        }
+    return user_data
```

Comment:
One way to refactor this so the movie is actually being added is `user_data["watched"].append(movie)`